### PR TITLE
enable "scripted" test framework

### PR DESCRIPTION
**Pull request by sullis**
details: https://www.scala-sbt.org/1.x/docs/Testing-sbt-plugins.html

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: src/sbt-test/sbt-guardrail/scala-client-codegen-app/test [hunk 1]
```diff
@@ -0,0 +1,2 @@
+> clean
+> package
```
Comment (blast-hardcheese) [inline src/sbt-test/sbt-guardrail/scala-client-codegen-app/test]: The most recent issue would have been caught by `; compile ; compile`, as already up-to-date files were not considered for compilation.

File: src/sbt-test/sbt-guardrail/scala-client-codegen-app/src/main/scala/Hello.scala [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package minproject
+
+object Hello {
+  def main(args: Array[String]) = {
+    println("Hello World!!")
```
Comment (blast-hardcheese) [inline src/sbt-test/sbt-guardrail/scala-client-codegen-app/src/main/scala/Hello.scala]: `Client.httpClient(Route.asyncHandler(Resources.routes(new Handler { ... }))).foo()` would be helpful to ensure the generated files are actually exercised.

**Comment by sullis** [comment]
Ready for review @blast-hardcheese

**Comment by blast-hardcheese** [comment]
Thank you!

**Pull request merged**